### DISCUSSION TITLE
Kulanka dhaqsoPay Team.

### DISCUSSION THREAD

**Discussion by sharafdin** [dugsiiyeinc/dhaqsoPay#91] [Announcements]
Habeen dambe waxaan yeelan doonna meeting kaas oo aan isku macsalaameena kana hadalno arimo bdn Insha'Allah! aan kaso wada qeyb galno. @dugsiiyeinc/helapi-contributors , @dugsiiyeinc/dhaqsopay-extension-developers , @dugsiiyeinc/dhaqsopay-server-developers

**Comment by iftiiin** [comment]
Diyaar eng

**Comment by hayle01** [comment]
Insha allah Eng
